### DISCUSSION TITLE
Deprecating Outdated Issues on the GitHub Public Roadmap

### DISCUSSION THREAD

**Discussion by ankneis** [github/roadmap#1014] [Announcements]
## **Deprecating Outdated Issues on the GitHub Public Roadmap**

At GitHub, transparency and clarity are at the heart of our relationship with the community. As part of our ongoing efforts to keep you informed about our product roadmap, we’ve already begun hosting [quarterly roadmap webinars](https://github.com/roadmap-webinar-series) to share updates and engage with the community in real-time. 

**This week, we’re taking the next step in achieving our roadmap goals by refreshing the public roadmap project board.**

After an in-depth review, we’ve identified a number of open issues that have become outdated over time—some for several years. To better align with our current product direction and to build trust with our users, we are deprecating these outdated issues and updating the board with new and accurate information.

This refresh will make it easier for you to follow our progress, ensure higher-quality updates, and provide a more accurate reflection of GitHub’s development priorities. Moving forward, we are also committing to regular updates, so you can rely on the roadmap as a trusted source of information about GitHub’s ongoing and upcoming features.

### **What’s Changing?**

* **Deprecation of Outdated Issues:** We will be removing a number of issues that are no longer relevant due to changes in priorities or project timelines. These issues have been stagnant and no longer represent our product direction.
* **Full Board Refresh:** A complete update to the roadmap board will be made, adding new features and plans that better reflect where GitHub is headed.
* **Ongoing Maintenance:** To ensure the roadmap stays accurate, we will be conducting regular reviews and updates.

### **FAQ**

**Why are we deprecating these issues?** 
 - Many of the issues marked for deprecation are out of date and no longer align with GitHub’s current roadmap. By cleaning up the board, we aim to provide more accurate, actionable insights to the community.

**What can I expect from the refreshed roadmap?**
 - The refreshed roadmap will feature up-to-date information about new initiatives and priorities. Our goal is to make the roadmap a reliable resource that users can check regularly for accurate and relevant details.

**Will the roadmap be updated regularly?**
 - Yes! We’re committing to regularly updating the roadmap to ensure it reflects the latest developments. This is part of our effort to build and maintain trust with our community.

**What should I do if an issue I care about is deprecated?**
 - While some issues may no longer be represented in the public roadmap, we are always listening to community feedback. If you have concerns about a specific feature or request, feel free raise your thoughts in the [discussions](https://github.com/orgs/community/discussions).

### **Deprecated Issues**

As part of this update, the following issues will be deprecated. If you have questions on a specific issue/roadmap item, please reach out to your GitHub contact. 

|Deprecated Issue|
|------|
|[Command Palette [GA]](https://github.com/github/roadmap/issues/355)| 
|[GitHub Actions: Secure Shell Debugging (Beta)](https://github.com/github/roadmap/issues/637)|
|[Actions: Managing Environments at Scale](https://github.com/github/roadmap/issues/988) |
|[GitHub Actions: Artifacts v4 available in GitHub Enterprise Server](https://github.com/github/roadmap/issues/930)|
|[Precise code navigation for Java](https://github.com/github/roadmap/issues/474)|
|[Precise code navigation for JavaScript and TypeScript](https://github.com/github/roadmap/issues/371)|
|[Issue level custom metadata](https://github.com/github/roadmap/issues/277)|
|[Ability to add a project to a project](https://github.com/github/roadmap/issues/274)|
|[Cross repository milestones and labels](https://github.com/github/roadmap/issues/276)|
|[Create custom automation flows](https://github.com/github/roadmap/issues/281)|
|[Projects: Activity History](https://github.com/github/roadmap/issues/816)|
|[Projects: Updated Projects header](https://github.com/github/roadmap/issues/761)|
|[[Public Beta] Issue Hierarchy powered by Tasklists](https://github.com/github/roadmap/issues/760)|
|[Packages: Granular permissions and easy organization sharing for enterprise customers](https://github.com/github/roadmap/issues/560)|
|[Packages: maven - granular permissions and easy organization sharing](https://github.com/github/roadmap/issues/578)|
|[GitHub Actions secrets improvements for Reusable workflows [GA]](https://github.com/github/roadmap/issues/636)|
|[More control over required status checks for pull requests using merge queue](https://github.com/github/roadmap/issues/824)|
|[Reply to PR-level comments](https://github.com/github/roadmap/issues/552)|
|[Codespaces: Pull Request Validation for prebuild-enabled repositories](https://github.com/github/roadmap/issues/537)|
|[Pull Request Dashboard](https://github.com/github/roadmap/issues/828)|
|[Dependabot on Actions Forced Migration](https://github.com/github/roadmap/issues/946)|
|[Secret scanning push protection for gists](https://github.com/github/roadmap/issues/969)|
|[Secret scanning: Extend coverage to Actions logs](https://github.com/github/roadmap/issues/449)|
|[Dependabot alerts shows transitive dependency paths](https://github.com/github/roadmap/issues/494)|
|[GHES Cluster High Availability](https://github.com/github/roadmap/issues/756)|
|[Open ID Connect (OIDC) for GHEC Audit Log Streaming to Azure Blob Storage](https://github.com/github/roadmap/issues/581)|
|[Azure AD (AAD) Service Principal Support for GHEC EMUs (Beta)](https://github.com/github/roadmap/issues/360)|
|[GitHub Actions Starter workflow improvements](https://github.com/github/roadmap/issues/772)|
|[Automatic security check information on each Actions listing](https://github.com/github/roadmap/issues/774)|
|[Actions: Private networking for GitHub-hosted macOS runners](https://github.com/github/roadmap/issues/982)|
|[Security manager improvements and custom organization security roles (Preview)](https://github.com/github/roadmap/issues/791)|
|[Expanding access to historical log data via audit log exports (Preview)](https://github.com/github/roadmap/issues/921)|
|[Code security transaction report in PDF](https://github.com/github/roadmap/issues/781)|
|[Code scanning: AI-powered autofixes for CodeQL alerts integrated into VS Code](https://github.com/github/roadmap/issues/916)|
|[Increase GitHub Enterprise Importer's (GEI) repository size limit to 40GB (Preview)](https://github.com/github/roadmap/issues/993)|
|[Enterprise Apps and installation automation (Public Beta)](https://github.com/github/roadmap-internal/issues/1423)|
|[GitHub Security Advisory private forks support Actions](https://github.com/github/roadmap/issues/627)|
|[GitHub Actions configuration variables - GA](https://github.com/github/roadmap/issues/773)|
|[Actions: Outbound network control for GitHub-hosted runners](https://github.com/github/roadmap/issues/821)|
|[Enterprise access for GitHub apps](https://github.com/github/roadmap/issues/753)|
|[Codespaces: Private networking with Azure VNETs (Preview)](https://github.com/github/roadmap/issues/534)|
|[Commenting on unchanged lines in a pull request](https://github.com/github/roadmap/issues/347#top)|

We appreciate your understanding as we make these changes. Our aim is to keep you better informed and involved in our development process. Thank you for being a valued member of the GitHub community!

---------------------------------------------------------------------------------------------------------------------
## 📰 Update: Nov 26, 2024

Hi GitHub Community! 👋

We know some of you are disappointed to see certain items removed from our public roadmap, and we really appreciate your feedback. Your input means a lot to us, and these changes reflect some tough prioritization decisions we’ve had to make to focus on delivering the most impactful solutions for everyone. 

We first want to apologize for a slightly misleading statement we made in our original post, that we hope to acknowledge and correct. We said that the closed issues _no longer represent our product direction_, however there are a number of reasons for removal, from the feature not being aligned with our strategic priorities, to our desire to actively acknowledge uncertainty on the timeline for a feature and remove it until we have more certainty.

In order to provide that missing clarity, we are adding more comments to the list of issues above with more details about why they were removed and whether there’s something similar on our refreshed roadmap. You’ll see those comments added over the next couple of weeks. While not everything will return to the roadmap, we’re dedicated to staying open, honest, and to regularly updating you. Look out for our next roadmap update in January—we’re excited to share the new possibilities we’re working on!

We invite you to keep the conversation going in our [Community](https://github.com/orgs/community/discussions/145255). It’s the best place to share your feedback, ask questions, and connect with our team and other users. Your thoughts play a big role in shaping what we do, and we’re here to listen and collaborate with you to make GitHub the best it can be.

**Comment by scamden** [comment]
https://github.com/github/roadmap/issues/552 is unbelievably painful UX. Gitlab has this. Not being able to thread on top level comments makes it so hard to track conversations and results in all this quotation noise. Please reconsider adding this feature (is it really that hard?? you already have threaded comments on lines..)

**Reply by alvgaona** [reply]
I wonder how this is not aligned to GitHub's roadmap.

**Reply by rjbell4** [reply]
I also cannot understand how this would be no longer relevant.

**Reply by fitz-vivodyne** [reply]
On what planet is #552 outdated?

**Reply by fitz-vivodyne** [reply]
FYI, I started a top level "Idea" discussion on #552 (threading for top level comments), see https://github.com/github/roadmap/discussions/1067

**Reply by ankneis** [reply]
We understand you're disappointed that #552 was removed from the roadmap. We've added more detail in a comment on the issue to provide additional context. Please stay tuned for additional updates to the roadmap in coming months and continue sharing your feedback on the roadmap over [here](https://github.com/orgs/community/discussions/145255) in Community!

**Comment by NatoBoram** [comment]
It's a little tone-deaf to call https://github.com/github/roadmap/issues/281 "outdated" when it's a daily struggle to manually perform the workflow we want automated. Or to call https://github.com/github/roadmap/issues/276 "outdated" when it's a daily struggle to sync labels and to work around milestones by using Linear in our org.

I also agree with https://github.com/github/roadmap/issues/552, a Twitter feed is unsuitable for pull request comments. They deserve their own replies.

I mean, it's great that you guys finally commit on not making the user experience better, but calling your top pain points "outdated for several years" is bound to not be well-received.

**Reply by jeherve** [reply]
> It's a little tone-deaf to call https://github.com/github/roadmap/issues/281 "outdated" when it's a daily struggle to manually perform the workflow we want automated.

It is a bit frustrating indeed. However, we are invited to continue the discussions in GitHub discussions if we're still interested in seeing some of those deprecated issues happen.

With that in mind, if you are as interested as I am to see custom workflows in Projects, I think those 3 discussions could be upvoted:

- https://github.com/orgs/community/discussions/45653
- https://github.com/orgs/community/discussions/53705
- https://github.com/orgs/community/discussions/53973

**Reply by Nimorid** [reply]
I am too really disappointed #276 was depreciated, I recently set-up a new Project and that was exactly what I missed and had to duplicate a lot of manual work.

**Reply by joshmanders** [reply]
"Outdated for several years"

Ya'll left it sit for several years, lmfao. I don't understand this mentality of "we haven't acknowledged or addressed this issue for several years, therefore we're going to mark it as stale and close it" that most people have.

**Reply by devnoname120** [reply]
IMO https://github.com/github/roadmap/issues/281 is better served by GitHub Actions. Implementing visual workflows would create an awkward situation where you can do things in two different ways, some features being available in the first but not in the second, and vice-versa.

This would make it frustrating to migrate from one to the other when you stumble upon missing features as well as induce decision paralysis. It would also mean less transparency to non-owner contributors of repos about the underlying workflows and prevent them from fixing them/doing a PR as needed.

**Reply by jeherve** [reply]
@devnoname120 That's probably something worth discussing in https://github.com/orgs/community/discussions/53973

GitHub actions definitely offer the most flexibility, but then comes the question of where should the code live for projects that are not tied to a specific repo.

**Reply by devnoname120** [reply]
@jeherve No it's something else. Compare the [schema of GitHub Workflows](https://github.com/SchemaStore/schemastore/blob/master/src/schemas/json/github-workflow.json) with the [schema of GitHub Composite Actions](https://github.com/SchemaStore/schemastore/blob/master/src/schemas/json/github-action.json). You will find out that there is a bunch of non-sensical[1] minute differences that makes everything unexpectedly break when you try to remove code duplication by refactoring it into GitHub Composite Actions.

[1] Non-sensical = there is absolutely no justification for the differences, they are arbitrary. For example in literally all the `run` steps of Composite Actions you need to write `shell: sh` or... it doesn't work because this property doesn't have a default. But in Workflows this **same exact property** has a default so you don't need to specify `shell: sh`.

**Reply by ankneis** [reply]
We hear your feedback and understand you're disappointed to see #281, #276, and #552 removed from the roadmap.  We have added more context in comments on each of those issues, and hope you'll continue sharing your feedback on the roadmap over [here](https://github.com/orgs/community/discussions/145255) in Community. And yes, @jeherve, we love it when people upvote suggestions in the Community.  :)

**Comment by connorjkennedy** [comment]
Myself and my organization were extremely interested in seeing #636 be implemented, as it would allow for a much tighter security posture around centralized reusable workflows.  At the current moment, one must share secrets with the repository consuming the reusable workflow, which is a blatant security risk, as anyone with write access to the calling repository can then cut their own branch, modify the action file, and use the secrets however they desire.   Are there any plans to implement this or similar functionality in the future?

**Reply by ViacheslavKudinov** [reply]
> Myself and my organization were extremely interested in seeing #636 be implemented, as it would allow for a much tighter security posture around centralized reusable workflows.  At the current moment, one must share secrets with the repository consuming the reusable workflow, which is a blatant security risk, as anyone with write access to the calling repository can then cut their own branch, modify the action file, and use the secrets however they desire.   Are there any plans to implement this or similar functionality in the future?

Same here.
Had a hope we will get it.
Sad to see it is no longer in the roadmap.

**Reply by marcosboger** [reply]
Same for our Team. It would save us a lot of time and make our centralized CICD framework more secure.

**Reply by reiniertimmer** [reply]
This would be an incredible security boost, as it will allow to tie secrets specifically to reusable workflows (as it now is using secrets from the calling repository).

This means that if you want to reuse workflows on many repositories, you also need to make sure that secrets are available on _all_ these repositories - and there is no way you can control _how_ these secrets are being used.

It will basically allow full freedom to use any secret in any way, whereas assigning it to a centrally controller repository, you can actually control how the secrets are being used.

As an alternative, we are considering using OIDC with `job_workflow_ref` to access a keyvault where the secrets are stored instead. This is quite a painful and slow workaround. It does work, but it is not ideal at all.

I would really like to see this feature in some form or another!

**Reply by ankneis** [reply]
Thank you for sharing this feedback. We have added more detail in a comment on #636 about the removal. Stay tuned for more updates to the roadmap in the coming months and continue to share your feedback in [Community](https://github.com/orgs/community/discussions/145255)!

**Comment by Anbu42** [comment]
We use GitHub Enterprise server, and our organization really needs support for [GitHub Actions: Artifacts v4](https://github.com/github/roadmap/issues/930)

Is there a chance this will be done? The cloud version has supported Artifacts v4 for a long time.

**Reply by wkjung** [reply]
I cannot believe supporting v4 is marked _outdated_ while v3 will be deprecated by **November 30, 2024**.

**Reply by Cube707** [reply]
yeah.

I recently ran into the following after running in circles for a while:

> Recommend switching to v4 as soon as possible! v1-v3 artifact still have these limitations

> download-artifact@v4+ is not currently supported on GHES yet. If you are on GHES, you must use [v3](https://github.com/actions/download-artifact/releases/tag/v3).

Made me want to run into a wall head first and I was really looking forward to getting v4 support for GHES

**Reply by shansrii** [reply]
GHES is cursed by not having Artifacts v4 yet and it (v3) is causing lot of noise in our developer community

**Reply by brookman** [reply]
v3 is one or two orders of magnitude slower at upload and download than necessary. I know because we have replaced it with Azure artifacts and Artifactory where possible (for intermediate artifacts). Still the majority of our build time is waiting until the final artifacts get uploaded…

**Reply by devnoname120** [reply]
It's ridiculous for GHES not to be kept on par with SaaS gitHub, making it harder to migrate from one to the other, and bad surprises popping up due to these inconsistencies. Differences/discrepancies between GitHub Workflows and GitHub Composite Actions are already direly painful, please don't make the situation worse by introducing additional footgun incongruences across GitHub.

**Reply by Danielku15** [reply]
If this is really intentional I am honestly considering to stop using GitHub Actions or build an own solution for artifacts. We migrated to GHA in GHES beginning this year and it has a pain ever since. We have upload and download times of up to 20-30minutes (a single large zip) with Gigabit connections, this is a pain and implies a lot of cost and risks on every build.

I really hope that it was "outdated" accidentally because they actually have no efforts internally for it and will enable usage in the next release.

**Reply by ankneis** [reply]
Thank you for this feedback regarding Artifacts v4 in GHES. We've added more detail in a comment on #930 and continue sharing your feedback on the roadmap over in the [Community](https://github.com/orgs/community/discussions/145255)!

**Comment by johnson-dave** [comment]
> #552 is unbelievably painful UX. Gitlab has this. Not being able to thread on top level comments makes it so hard to track conversations and results in all this quotation noise. Please reconsider adding this feature (is it really that hard?? you already have threaded comments on lines..)

I agree with this too - quote-replies instead of threaded replies provide an awful experience.

Example: This quote reply.

**Reply by christophe-kamphaus-jemmic** [reply]
Btw, discussions have reply feature.
Why not on PRs?

**Reply by devnoname120** [reply]
This is the reason issues and PRs garner 500 comments and are impossible to follow in this linear feed. GitHub massively collapsing hundreds of comments makes it even worse because you can't even search in this mess to find the information you're looking for. I've been starting to disable GitHub Issues entirely from my projects and shoehorn Discussions into a makeshift issue tracker. Very awkward.

**Reply by ankneis** [reply]
Thank you for providing your feedback on #552 - we've added more detail in a comment on the issue. Please stay tuned for additional updates to the roadmap in coming months and continue sharing your feedback on the roadmap over [here](https://github.com/orgs/community/discussions/145255) in Community!

**Comment by FrancescoBonizzi** [comment]
Also #824 (More control over required status checks for pull requests using merge queue) has been abandoned. Sad. 

This is a common use case where certain PR checks should be skipped based on specific filters.

In our monorepo, we have three main projects:
- Mobile App
- Web
- Backend

We use three PR action checks, each filtered by folder. As a safety measure, it’s crucial to block merging into the main branch if the relevant check hasn’t passed. However, without this feature, we’re forced to leave all checks optional, which compromises safety.

**Reply by DouglasBlackwood** [reply]
GitHub's merge queue is quite basic and it's meant to stay like that. They haven't invested much in it since its release last year.

Did you have a look into advanced merge queues, like [Mergify](https://docs.mergify.com/merge-queue/setup/)? Some of them have dedicated features for monorepos.

**Reply by FrancescoBonizzi** [reply]
Thanks, I'm looking into it!

**Reply by ankneis** [reply]
Thank you for sharing your feedback. We've added additional detail in a comment on #824 and and continue sharing your feedback on the roadmap over [here](https://github.com/orgs/community/discussions/145255) in Community!

**Comment by mbrevda** [comment]
So your closing issue without discussion, reasoning, or explanation - all in the name of transparency?

**Reply by beedell-roke** [reply]
> https://github.com/github/roadmap/discussions/1014#discussioncomment-11328203

@mbrevda, all but one were already locked. [^1]

[^1]: https://github.com/github/roadmap/issues/828#event-15385186485

**Reply by hyperTwitch** [reply]
Some of these were also locked without any discussion, reasoning, or explanation.

**Reply by beedell-roke** [reply]
> https://github.com/github/roadmap/discussions/1014#discussioncomment-11351191

@hyperTwitch, which?

**Reply by hyperTwitch** [reply]
Every single ticket I can see in this list is locked and has zero discussion, at least that the public is able to see. Just a post saying that the ticket is now "outdated" and is being closed. As you can see from the discussion here, there are more than a couple tickets on this list which the community does not understand how the feature request is "outdated".

**Reply by beedell-roke** [reply]
> https://github.com/github/roadmap/discussions/1014#discussioncomment-11351337

@hyperTwitch, there is https://github.com/github/roadmap/issues/828#issuecomment-2489168831... but that's about it XD

**Reply by ankneis** [reply]
Thank you for flagging this and we understand your frustration. We have been working on adding details to the issues we've removed from the roadmap and regret not publishing that additional context to you sooner. Please check out the comments we're adding with more details on the issues you're interested in, and we're continuing to add more context and details through early December. We encourage you to continue sharing your feedback about the roadmap [in this post](https://github.com/orgs/community/discussions/145255), or feel free to open up a new thread on the [Community](https://github.com/orgs/community/discussions) with any specific questions or concerns you have!

**Comment by moattarwork** [comment]
> https://github.com/github/roadmap/issues/988

It is very disrespectful of GitHub to kill the roadmap items without explaining what is wrong with supporting enterprise-wide environments and make you stupidly define environments in multiple repo. Especially with microservices implementation, someone should be out of mind to choose Github actions

**Reply by tetienne** [reply]
I can only agree on this. It would have allowed us to easily enforce some rules on any new repository. That’s a shame.

**Reply by ankneis** [reply]
We appreciate your feedback and acknowledge your disappointment with the removal of #988 from the roadmap. We've added an additional comment on the issue to provide context. Please stay tuned for additional updates to the roadmap in coming months! [Join our community](https://github.com/orgs/community/discussions) to continue the conversation.

**Comment by h-m-f-t** [comment]
Thanks for this update, @ankneis. 

My team at the .gov registry has been really pleased with sub-issues, and I was hoping to see updates on the 3 issues below! Several of our repos feed into a [single project](https://github.com/orgs/cisagov/projects/26), and being able to share cross-repo milestones and labels, track dependencies, and review our project history for errant changes would be so useful to our delivery.
- #276
- #956
- #816

Prioritization decisions are hard! If my team can ever be useful in your discovery and research efforts, we'd love to be.

**Reply by julienvincendeau-rc** [reply]
Totaly agree with you
I'm starting looking Gitlab beacause of that 

Please wakeup @github-product-roadmap

**Reply by ankneis** [reply]
We hear your feedback and understand you're disappointed to see #276, #956, and #816 removed from the roadmap.  We have added more context in comments on each of those issues, and hope you'll continue sharing your feedback on the roadmap over [here](https://github.com/orgs/community/discussions/145255) in Community. Stay tuned for another roadmap update in January!

**Comment by hauntsaninja** [comment]
I run up against https://github.com/github/roadmap/issues/347 regularly, would love for Github to add it!

One downside of getting rid of these issues is that there's now no easy way to be alerted when a feature I care about has been implemented

**Reply by ankneis** [reply]
We hear your feedback around #347, @hauntsaninja. We've added more context on #347 and stay tuned for more roadmap updates in the coming months.  [Join our community](https://github.com/orgs/community/discussions/145255) to continue the conversation.

**Comment by styfle** [comment]
Please add this to your roadmap:

- https://github.com/github/roadmap/issues/347

Frankly, I don't understand why this hasn't been prioritized. Does GitHub not dogfood their own product? How are the GitHub engineers reviewing each other's code?

**Reply by rjbell4** [reply]
We have hundreds of users that are refusing to use GitHub pull request reviews because of this issue.  We'll definitely be raising this with our account team.

**Reply by dufcrule** [reply]
A sad day for reviews 😞.

**Reply by justjake** [reply]
Graphite helps

**Reply by ankneis** [reply]
Thank you @styfle (and those replied) for sharing this feedback and we understand the disappointment around the removal of this item from the roadmap. We have added more detail in a comment on #347. Please stay tuned for more updates to the roadmap in the coming months and continue sharing your feedback on the roadmap over [here](https://github.com/orgs/community/discussions/145255) in the Community!

**Comment by chrishengler** [comment]
Commenting unchanged lines is something I've run into relatively rarely, but when it has happened the workaround of "leave a comment on some random changed line and explain where it actually applies to" has been extremely frustrating whether as reviewer or as reviewee.

The lack of threaded replies for top-level PR comments was always a baffling design decision, especially when they exist on other kinds of comment and in discussions.

Very disappointed to see #347 and #552 off the roadmap, both would be major usability improvements.

**Reply by ankneis** [reply]
We appreciate your feedback and understand you're disappointed to see #347 and #552 removed from the roadmap. We've added more detail to both of those issues and will continue to update our roadmap in the coming months. Feel free to continue the conversation [here](https://github.com/orgs/community/discussions/145255) on the Community!

**Comment by Crosswind** [comment]
I am confused why #930 was removed from the roadmap. Dependabot is asking us to upgrade to v4 in GHES while it remains incompatible. Does the removal of the issue mean GHES will never see support for articat-upload/-download@v4 in GHES?

**Reply by ankneis** [reply]
Thank you for sharing your feedback. We've added additional detail in a comment on #930. Feel free to continue sharing your feedback on the roadmap over [here](https://github.com/orgs/community/discussions/145255) in Community!

**Comment by oliversalzburg** [comment]
Thank God that you kept the Copilot features on the agenda, and stripped some of this outdated security and user experience stuff. I'm sure this will help me to rely more on your AI offerings, instead of getting work done myself. Glad you realigned with your current product vision. Having this transparently communicated really helps to see that vision unfold.

**Comment by BerntJulian** [comment]
Packages: maven - granular permissions and easy organization sharing  https://github.com/github/roadmap/issues/578 Is a necessity if one want so use maven packages on a organization level.

**Reply by ankneis** [reply]
Thank you for sharing your feedback. We've added additional detail in a comment on #578. Jump into the community [here](https://github.com/orgs/community/discussions/145255) to continue sharing your feedback on the roadmap.

**Comment by GoldsteinE** [comment]
To save someone time from clicking on all the “additional detail” comments: as far as I can see, they’re all basically “we don’t know when we’re going to do this” and in some cases “we don’t know whether we’re going to do this at all”.

Example phrasing:

> We wanted to provide more details on why this issue was removed from the public roadmap. We intend to provide activity history for Projects, but we're removing it from this roadmap for now until we have more certainty on our timeline. Please keep an eye out for a new roadmap issue coming soon with more details!

**Comment by ankneis** [comment]
We have made some updates to our Discussion post in an aim to provide additional context and clarity. 

We are adding more information and details to the removed issues linked above, some of which have new or related issues being added to the roadmap with up-to-date descriptions. We value your feedback on these changes and invite you to continue the conversation [here](https://github.com/orgs/community/discussions/145255) in our community discussion, where we’ll also be posting updates about the new features we’re adding to the roadmap - it’s the best place to share your thoughts, ask questions, and connect with us and other users.
